src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,117 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TextUi;
+import duke.task.TaskList;
+
+import java.io.IOException;
+
+/**
+ * The Command class is used to pass commands around the Duke application
+ * once user input has been parsed.
+ */
+public abstract class Command {
+    /** Details of the command. */
+    protected String details;
```

Comment:
Perhaps the meaning of "details" could be better documented? I had to check the uses of this field in the concrete class to verify that it refers to the arguments of the Command.